File: PythonStmSrv/mainSystem.py
from server import ServerStm
import sys

if __name__ == "__main__":
    #while True:
    if len(sys.argv) > 3:
        ServerStm(sys.argv[1], sys.argv[2], sys.argv[3])
    else:
        ServerStm("localhost", "myRequest", "myResponse")

File: JavaJmsSrv/src/main/java/mainSystem.java
public class mainSystem {
    public static void main(String[] args) {
        try {
            if (args.length > 2) {
                new Server(args[0], args[1], args[2]);
            } else {
                new Server("localhost", "myRequest", "myResponse");
            }
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}
